cmd/fyne_demo/screens/widget.go

```diff
@@ -30,6 +29,14 @@ func makeButtonTab() fyne.Widget {
 	disabled := widget.NewButton("Disabled", func() {})
 	disabled.Disable()
 
+	return widget.NewVBox(
+		widget.NewButton("Button (text only)", func() { fmt.Println("tapped text button") }),
+		widget.NewButtonWithIcon("Button (text & icon)", theme.ConfirmIcon(), func() { fmt.Println("tapped text & icon button") }),
+		disabled,
+	)
+}
+
+func makeTextGrid() *widget.TextGrid {
```

Comment:
You could expand this TextGrid demo if you wanted to show it off more (two lines is a little small, especially if it follows Lorem Ipsum)

Reply:
True. I will update that in a different commit where TestGrid gets some love